aws-iot-mitigationaction/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,15 @@
+{
+  "ActionName": "CfnContractTest",
```

Comment:
It will be nice to have a test for every single action that we support, else how do you know that the definition of one of the actions isn't broken?

Reply:
Adding an integ test for them in the integ test package.